python/ray/data/grouped_dataset.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, dataset: Dataset[T], key: GroupKeyT):
         else:
             self._key = key
 
-    def aggregate(self, *aggs: Tuple[AggregateFn]) -> Dataset[U]:
+    def aggregate(self, *aggs: AggregateFn) -> Dataset[U]:
```

Comment:
aggs is Tuple?

Reply:
The convention is to annotate variadic arguments with the type of the individual arguments; Python will deduce `aggs: Tuple[AggregateFn]` in the body of the function. https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values

Reply:
Then lets fix the typing for `dataset.aggregate` as well

Reply:
Ah I missed that, good catch!